src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -463,7 +459,7 @@ export default {
 			return this.totalCount <= this.limit;
 		},
 		showHelpMeChooseFeat() {
-			return this.enableHelpmeChoose && this.allLoans.length > 8;
+			return this.allLoans.length > 8;
```

Comment:
This will conflict with my PR, go ahead and merge this first once it's been approved.